apis/python/src/tiledb/vector_search/embeddings/langchain_embedding.py

```diff
@@ -37,7 +40,10 @@ def vector_type(self) -> np.dtype:
     def load(self) -> None:
         import importlib
 
-        embeddings_module = importlib.import_module("langchain.embeddings")
+        if self.use_langchain_openai:
```

Comment:
Instead of having the user pass a parameter can we do an import and try/except to handle this automatically?

See example at: https://github.com/TileDB-Inc/TileDB-ML/blob/effa1c1c7a80026540ce157d4d01c47207ce5ad7/tests/models/test_tensorflow_keras_models.py#L19

Reply:
At the moment this is just a warning, will also need an option to load embeddings from `langchain_community`. Should I just load from the new modules?

Reply:
I'm fine with setting minimum versions or additional package requirements to keep this as simple as possible.

Reply:
Applied Seth's suggestion with a preference to first try to import from `langchain_openai`